.github/workflows/notebooks-digest-updater-upstream.yaml

```diff
@@ -89,6 +89,16 @@ jobs:
               done
               git fetch origin  ${{ env.DIGEST_UPDATER_BRANCH }} && git pull origin  ${{ env.DIGEST_UPDATER_BRANCH }} && git add manifests/base/params.env && git commit -m "Update images for release N via ${{ env.DIGEST_UPDATER_BRANCH }} GitHub action" && git push origin  ${{ env.DIGEST_UPDATER_BRANCH }}
 
+              name: Fetch digest, and update the commit.env file
+              run: |
+                    echo Latest commit is: ${{ steps.hash-n.outputs.HASH_N }} on ${{ env.RELEASE_VERSION_N}}
+                    IMAGES=("odh-minimal-notebook-image-commit-n" "odh-minimal-gpu-notebook-image-commit-n" "odh-pytorch-gpu-notebook-image-commit-n" "odh-generic-data-science-notebook-image-commit-n" "odh-tensorflow-gpu-notebook-image-commit-n" "odh-trustyai-notebook-image-commit-n")
+                    for ((i=0;i<${#IMAGES[@]};++i)); do
+                      image=${IMAGES[$i]}
+                      echo HASH_N
+                    done
+                    git fetch origin  ${{ env.DIGEST_UPDATER_BRANCH }} && git pull origin  ${{ env.DIGEST_UPDATER_BRANCH }} && git add manifests/base/commit.env && git commit -m "Update image commits for release N via ${{ env.DIGEST_UPDATER_BRANCH }} GitHub action" && git push origin  ${{ env.DIGEST_UPDATER_BRANCH }}
```

Comment:
This has been a bash code, 
name and run, shouldn't be part of it.
Seems like a typo, please correct it.

Another suggestion, 
As the goal is to update the commit hash in the `commit.env` file.

Use a separate var `COMMIT`, to not disrupt the image follow.
```
COMMIT=("odh-minimal-notebook-image-commit-n" "odh-minimal-gpu-notebook-image-commit-n" "odh-pytorch-gpu-notebook-image-commit-n" "odh-generic-data-science-notebook-image-commit-n" "odh-tensorflow-gpu-notebook-image-commit-n" "odh-trustyai-notebook-image-commit-n")
```
utilize a loop over this list.
```
for val in "${COMMIT[@]}"; do
 echo $val
 sed -i "s|${val}=.*|${val}=${{ steps.hash-n.outputs.HASH_N }}|" manifests/base/commit.env
done
```
this would patch the values with correct hash.

we could directly include this in above part , so we don't have to do git operation twice.
WDYT?